util/version/version.go

```diff
@@ -229,6 +213,14 @@ func Compare(a, b semver.Version, options ...CompareOption) int {
 	}
 
 	if c.buildTags {
+		CompareWithBuildIdentifiers := func(a semver.Version, b semver.Version) int {
+			if comp := a.Compare(b); comp != 0 {
+				return comp
+			}
+			biA := newBuildIdentifiers(a.Build)
+			biB := newBuildIdentifiers(b.Build)
+			return biA.compare(biB)
+		}
 		return CompareWithBuildIdentifiers(a, b)
```

Comment:
We can inline the func like that and thus avoid defining a separate func
```suggestion
		if comp := a.Compare(b); comp != 0 {
			return comp
		}
		biA := newBuildIdentifiers(a.Build)
		biB := newBuildIdentifiers(b.Build)
		return biA.compare(biB)
```